docs/python/debugging.md

```diff
@@ -407,6 +407,8 @@ Specifies arguments to pass to the Python program. Each element of the argument
 
 If you want to provide different arguments per debug run, you can set `args` to `${command:pickArgs}`. This will prompt you to enter arguments each time you start a debug session.
```

Comment:
```suggestion
If you want to provide different arguments per debug run, you can set `args` to `"${command:pickArgs}"`. This will prompt you to enter arguments each time you start a debug session.
```